src/admin/content-block/content-block.const.ts

```diff
@@ -80,9 +91,19 @@ export const CONTENT_BLOCK_INITIAL_STATE_MAP = {
 	[ContentBlockType.IFrame]: INITIAL_IFRAME_BLOCK_COMPONENT_STATE,
 };
 
-// Heading
+// Options
 export const HEADING_LEVEL_OPTIONS: SelectOption<HeadingLevelOptions>[] = [
 	{ label: 'H2', value: 'h2' },
 	{ label: 'H3', value: 'h3' },
 	{ label: 'H4', value: 'h4' },
```

Comment:
mss moeten we deze ook vertaalbaar maken zodat ze indien ze willen de labels kunnen updaten naar:
* Groot (H2)
* Medium (H3)
* Klein (H4)